frameworks/Swift/vapor/vapor.dockerfile

```diff
@@ -4,7 +4,7 @@ ADD ./ /vapor
 WORKDIR /vapor
```

Comment:
is it possible to build from this docker file: https://hub.docker.com/_/swift/? Not sure what TFB policy is there, but not needing to install swift would save time.

Reply:
Aren't the dockerfiles already using the last stable version ? (`FROM swift:4.1`)